crates/matrix-sdk-base/src/read_receipts.rs

```diff
@@ -29,24 +29,16 @@ impl PreviousEventsProvider for () {
     }
 }
 
-#[instrument(skip_all, fields(room_id))]
+#[instrument(skip_all, fields(room_id = %room_info.room_id))]
 pub(crate) async fn compute_notifications(
     client: &BaseClient,
     changes: &StateChanges,
     previous_events_provider: &dyn PreviousEventsProvider,
     new_events: &[SyncTimelineEvent],
     room_info: &mut RoomInfo,
 ) -> Result<()> {
-    // Only apply the algorithm to encrypted rooms, since unencrypted rooms' unread
-    // notification counts ought to be properly computed by the server.
-    if !room_info.is_encrypted() {
-        return Ok(());
-    }
```

Comment:
Do we apply this function on non-encrypted room too? It means the value from the server is **always** ignored? It should be specified in the documentation.